lib/get/get-full-source-space.js

```diff
@@ -94,5 +94,5 @@ function extractItems (response) {
 }
 
 function filterDrafts (items, includeDrafts) {
-  return includeDrafts ? items : items.filter((item) => !item.isPublished())
+  return includeDrafts ? items : items.filter((item) => !item.sys.publishedVersion)
```

Comment:
the logic is right because the `items` array includes drafts by default, we filter it if the user does not specify `--include-drafts`